onelogin/get.go

```diff
@@ -90,26 +92,7 @@ func Get(app, provider string) (*awsprovider.Credentials, error) {
 	st := rSaml.Data[0].StateToken
 
 	devices := rSaml.Data[0].Devices
-
-	var deviceID string
-	var deviceType string
-
-	if len(devices) > 1 {
-		for i, d := range devices {
-			fmt.Printf("%d. %d - %s\n", i+1, d.DeviceId, d.DeviceType)
-		}
-
-		fmt.Printf("Please choose an MFA device to authenticate with (1-%d): ", len(devices))
-		var selection int
-		fmt.Scanln(&selection)
-
-		deviceID = fmt.Sprintf("%v", devices[selection-1].DeviceId)
-		deviceType = devices[selection-1].DeviceType
-
-	} else {
-		deviceID = fmt.Sprintf("%v", devices[0].DeviceId)
-		deviceType = devices[0].DeviceType
-	}
+	deviceID, deviceType, err := getDevice(devices)
```

Comment:
`getDevice()` should now return a `*Device` since we added this type. No point in returning unstructured strings. Also, errors should **always** be checked. I suggest the following:

```go
d, err := getDevice(devices)
if err != nil {
    return nil, fmt.Errorf("selecting MFA device: %v", err)
}
```